build/debug_keybase/main.go

```diff
@@ -67,7 +66,7 @@ func dumpKeybase(privateKeysYamlBytes []byte, targetFilePath string) {
 	defer os.RemoveAll(tmpDir)
 
 	kb, err := keybase.NewKeybase(&configs.KeybaseConfig{
-		FilePath: defaults.DefaultRootDirectory + "/keys",
+		FilePath: tmpDir,
```

Comment:
@h5law `tmpDir` above seems to be created for that purpose and it is not used anywhere else. Please lmk if you think that's not the case - we can remove `tmpDir` and keep `FilePath` pointed to `defaults.DefaultRootDirectory + "/keys"`.

Reply:
I think we need to point it to the `defaults.DefaultRootDirectory + "/keys"` as this means on a fresh install it will not only create the backup dump of keys but also fill the keybase in the default location. Otherwise using the `tmpDir` we will create the dump but it will never actually be loaded into where this is usable.

Probs just remove L66 and keep the default path as the `FilePath` field is best IMO, wdyt?

Reply:
@h5law makes sense, thank you!